lib/postprocessing/image.js

```diff
@@ -233,10 +294,16 @@ async function needsImagePostProcess(rendition, source) {
         return false;
     }
 
-    await logImageMagickVersion();
-
     // inspect intermediate rendition
-    const metadata = rendition._intermediateMetadata = await getImageMetadata(rendition.path);
+    const metadata = await getImageMetadata(rendition.path);
+    rendition._intermediateMetadata = metadata;
+
+    if(!metadata.FileType){
+        // could not recognize filetype - most likely it is corrupt
+        throw new SourceCorruptError(`Rendition seems to be corrupt. Could not identify file type.`);
```

Comment:
We can now detect additional cases in which the source file seems to be corrupt.